libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -91,7 +91,7 @@ const AP_Param::GroupInfo AP_Logger::var_info[] = {
     // @Param: _DISARMED
     // @DisplayName: Enable logging while disarmed
     // @Description: If LOG_DISARMED is set to 1 then logging will be enabled at all times including when disarmed. Logging before arming can make for very large logfiles but can help a lot when tracking down startup issues and is necessary if logging of EKF replay data is selected via the LOG_REPLAY parameter. If LOG_DISARMED is set to 2, then logging will be enabled when disarmed, but not if a USB connection is detected. This can be used to prevent unwanted data logs being generated when the vehicle is connected via USB for log downloading or parameter changes.
-    // @Values: 0:Disabled,1:Enabled,2:Disabled on USB connection
+    // @Values: 0:Disabled,1:Enabled,2:Disabled on USB connection,3:Disabled 1st disarmed
```

Comment:
probably rename to continue after arm

Reply:
start after armed      is what it does, correct? and continues after disarm until power removed?

Reply:
Yes

Reply:
Japan's drone flight time is defined as the time from the first takeoff until the power is turned off.  If you move to another location without turning off the arm, this time will also count as flight time.

Reply:
the option should be called "Start after arming"

and to @peterbarker comment, I agree this can be simply done with the mavlogdump.py utility to trim the normal log disarmed (although using the dont log until battey connect option should be recommended to shorten the log) before arming occurs. Could even make that an option in the mavlogdump tool so the condition does not need to be specified and I am sure it would be trivial for any python god, like Peter, to do in a few minutes. ie
mavlogdump.py <log> --trim-to-arm    
this avoids any flash usage